src/platform/mt8195/include/platform/drivers/mt_reg_base.h

```diff
@@ -0,0 +1,179 @@
+/* Copyright Statement:
+ *
+ * This software/firmware and related documentation ("MediaTek Software") are
+ * protected under relevant copyright laws. The information contained herein
+ * is confidential and proprietary to MediaTek Inc. and/or its licensors.
```

Comment:
Looks like this is not confidential any more...

Reply:
I miss those part, I will correct this.